test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/section/PrerequisiteSection.java

```diff
@@ -295,6 +296,17 @@ static void parseSkipSection(XContentParser parser, PrerequisiteSectionBuilder b
         parser.nextToken();
     }
 
+    @UpdateForV9
+    private static boolean parseRestCompatVersion(XContentParser parser, PrerequisiteSectionBuilder builder) throws IOException {
+        // allow skip version only for v7 REST compatibility tests, to be removed for V9
+        if ("true".equals(System.getProperty("tests.restCompat"))) return parseString(parser, builder::skipIfVersion);
```

Comment:
super nit: I find ifs without `{` `}` very confusing to read (but I know this might me *my* problem :) )